libs/core/src/global/styles/tokens/tokens.json

```diff
@@ -0,0 +1,437 @@
+{
+  "sage": {
+    "color": {
+      "white": {
+        "white": {
+          "value": "#ffffff",
+          "type": "color",
+          "description": "Backgrounds, primary/destructive CTA text"
+        }
+      },
+      "black": {
+        "black": {
+          "value": "#000000",
```

Comment:
No preference really, but should we use the shorthand hex notation where possible?

Reply:
I've updated this for now. Previously, I had issues with linting rules, but it doesn't seem to be a problem at the moment.